packages/fxa-payments-server/tsconfig.json

```diff
@@ -13,7 +17,9 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "preserve"
```

Comment:
The new test setup that runs both frontend and server-side tests needs this `"jsx": "react"` setting, if I recall.

Reply:
Oh, I see, but other bits from create-react-app (i.e. `react-scripts build`) want to switch it back :/ Might need to do some more headscratching on this

Reply:
I'll change it back.